be/src/exec/pipeline/operator.h

```diff
@@ -261,6 +263,10 @@ class Operator {
     // memory to be reserved before executing set_finishing
     virtual size_t estimated_memory_reserved() { return 0; }
 
+    // if return true it means the operator has child operators
+    virtual bool is_combinatorial_operator() const { return false; }
```

Comment:
according to your name and comment, MultilaneOperator should also be true.

Reply:
yes. this we need implements it for MultiLanceOperator/ConjugateOperator later.